docker/README.rst

```diff
@@ -0,0 +1,173 @@
+Docker Setup for FermiLib + ProjectQ
+====================================
+
+This Docker image will help users to easily install `FermiLib <https://github.com/ProjectQ-Framework/FermiLib.git>`__ and `ProjectQ <https://github.com/ProjectQ-Framework/ProjectQ>`__. Check out Docker's `website <https://www.docker.com/what-container>`__ that describes what a container image is and why it can be so useful.
+
+What is included?
+-----------------
+
+- Python 2.7 (you can also use Python 3 with one minor change in the Dockerfile. See the Dockerfile for instructions.)
+- `ProjectQ <https://github.com/ProjectQ-Framework/ProjectQ.git>`__ 
+- `FermiLib <https://github.com/ProjectQ-Framework/FermiLib.git>`__
+
+How to use it?
+--------------
+
+Installing and running Docker
+-----------------------------
+
+1. Install Docker from the `Docker website <https://docs.docker.com/engine/installation/#supported-platforms>`
+
+2. When Docker is installed, open a command line terminal and check the list of
+running virtual machines by
+
+.. code-block:: bash
+	docker-machine ls
+
+The returned list should be empty if this is the first time Docker is run on
+the computer. Create a virtual machine by running
+
+.. code-block:: bash
+	docker-machine create --driver virtualbox default
+
+(Note: To be able to run this, one needs to install `virtualbox <https://www.virtualbox.org/wiki/Downloads>`)
+
+Here "default" is just the name of the virtual machine. You can replace it by
+any name that you prefer. To check that the virtual machine is running indeed,
+use `docker-machine ls` again.
+
+3. When the Docker virtual machine is created, configure the shell by running
+
+.. code-block:: bash
+	docker-machine env default
+
+where if you named the virtual machine differently from default you should also
+replace "default" with the customized name. The command above will return a
+message containing the command to run for configuring the shell. This command
+depends on the OS.
+
+4. Run the command in the message returned above.
+
+
+Running OpenFermion with Docker
+-------------------------------
+
+5. Now that Docker is set up, one could navigate to the folder containing the
+Dockerfile for building the OpenFermion image (docker/dockerfile) and run
+
+.. code-block:: bash
+	docker build -t kickass_openfermion .
```

Comment:
While I appreciate the humor here, perhaps do not call it this. OpenFermion is a family friendly package!

Reply:
fixed